### PR TITLE
Special support for IE to workaround cross-origin popups

### PR DESCRIPTION
**Pull request by dvoytenko**
In a nutshell: in IE the popup is open as `about:blank` and then redirected to the final URL.

### CODE REVIEW

File: src/activity-window-port.js [hunk 1]
```diff
@@ -173,15 +173,45 @@ export class ActivityWindowPort {
...
+    // IE often returns `null` from the `window.open` API for a cross-origin
+    // target URL, making it impossible to set up messaging. The best path is
+    // to first open a blank window and then replace its location to the target
+    // url. This, however, needs to be done very carefully - if the browser
+    // instead forces redirect to "about:blank", the page's JS context will be
+    // destroy and it may fail to change the location to the target URL.
+    // As such, this option is only currently implemented for IE.
+    if (openTarget != '_top') {
+      const nav = this.win_.navigator;
+      const openViaBlank = /MSIE/i.test(nav && nav.userAgent);
```
Comment (kaipakartik) [inline src/activity-window-port.js]: leave a comment here that MSIE represents IE?

Comment (dvoytenko) [reply src/activity-window-port.js]: There's mentioning of it two lines above. Do you think another one needed here as well?

**Comment by kaipakartik** [comment]
MSIE referring to IE is the one I was referring to.

It's not obvious that IE has MSIE in the user agent string

File: src/activity-window-port.js (hunk 1 above)
Comment (dvoytenko) [reply src/activity-window-port.js]: SG. Added the comment. PTAL.

**Comment by dvoytenko** [comment]
Unfortunately, this solution is not fully feasible. Tests show that this is not fully possible. There are indeed still some workarounds, but they are too drastic, non-secure and likely not worth the effort for IE at this time. I'm closing this PR and instead sending #38 which is much simpler and reliable. Fully realizing that this would preclude some products from being launched on IE, but the alternatives are not secure enough either.

**Pull request closed without merging**